packages/salesforcedx-vscode-soql/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "salesforcedx-vscode-soql",
+  "displayName": "SOQL",
+  "description": "Provides code-editing features for SOQL",
+  "qna": "https://github.com/forcedotcom/salesforcedx-vscode/issues",
+  "bugs": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode/issues"
+  },
+  "repository": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode"
+  },
+  "version": "0.0.1",
+  "publisher": "salesforce",
+  "license": "BSD-3-Clause",
+  "engines": {
+    "vscode": "^1.46.0"
+  },
+  "categories": [
+    "Programming Languages"
+  ],
+  "dependencies": {
+    "@salesforce/soql-builder-ui": "0.0.2"
+  },
+  "devDependencies": {
+    "@types/node": "12.0.12",
+    "@types/vscode": "^1.46.0",
+    "typescript": "3.7.5"
+  },
+  "scripts": {
+    "compile": "tsc -p ./",
+    "lint": "tslint --project .",
```

Comment:
We'll be moving to using `eslint` soon since `tslint` is deprecated so it'd be good for you to start by using eslint. You could directly add it for this module only for now.

Reply:
ok, I see the use of ` "eslint": "6.0.1"` at the top level and one `eslint.json`.
Is that version ok and should I just copy the same config values?